code/html/work.html

```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Arbeiten</title>
+  <link rel="stylesheet" href="../style.css" />
+
+</head>
+
+<body>
+  <header>
+
+    <nav id="hamburger">
+      <div></div>
+      <div></div>
+      <div></div>
+    </nav>
+
+    <nav id="navbar">
+      <a href="./work.html" class="pagelink">Arbeiten</a>
+      <a href="./about.html">Über uns</a>
+      <a href="./contact.html">Kontakt</a>
+    </nav>
+
+    <a href="../index.html">
+      <div id="logo">
+        salami <br>hoch <br>drei
+      </div>
```

Comment:
I really like your logo! The `<br>`-tag though, is not used very often anymore. Another way to create the same thing in a more "modern" way would be to use divs, like you did in the hamburger, and style it using flexbox. 😊